### PR TITLE
add function: system sync file cache(sync syscall)

### PR DESCRIPTION
**Pull request by freedomDR**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
add **SYSTEM SYNC FILE CACHE** command. It will call sync syscall. It achieve [#8921](https://github.com/ClickHouse/ClickHouse/issues/8921)

### CODE REVIEW

File: src/Access/Common/AccessType.h [hunk 1]
```diff
@@ -171,6 +171,7 @@ enum class AccessType
     M(SYSTEM_WAIT_LOADING_PARTS, "WAIT LOADING PARTS", TABLE, SYSTEM) \
     M(SYSTEM_SYNC_DATABASE_REPLICA, "SYNC DATABASE REPLICA", DATABASE, SYSTEM) \
     M(SYSTEM_SYNC_TRANSACTION_LOG, "SYNC TRANSACTION LOG", GLOBAL, SYSTEM) \
+    M(SYSTEM_SYNC_FILE_CACHE, "SYNC SYSCALL", GLOBAL, SYSTEM) \
```
Comment (alexey-milovidov) [inline src/Access/Common/AccessType.h:174]: Let's name it exactly as the command.

**Comment by alexey-milovidov** [comment]
Need to update two test references.

File: src/Access/Common/AccessType.h (hunk 1 above)
Comment (freedomDR) [reply src/Access/Common/AccessType.h]: ok.

**Comment by freedomDR** [comment]
> Need to update two test references.

Update. It's having one failed tests, It doesn't matter.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**